.goreleaser.yml

```diff
@@ -0,0 +1,17 @@
+project_name: pidpath
+
+before:
+  hooks:
+    - go mod tidy
+
+builds:
```

Comment:
uuh, do you think we could also include `windows`? I noticed you included `pidpath_windows`, it'd be cool if we could also distribute a binary for that _os_ 😁

Reply:
yeah, thought about it too! truth is I've never dabble in Windos cross compilation or potentially Windows Docker environments. but tbh I've also never tried or researched it. I'll take this as an excuse to change that~

Reply:
TIL, at least when no platform-specific C libraries are required, Go can simply cross-compile 🤦